src/Inkluzitron/Models/Settings/BdsmTestOrgSettings.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+
+namespace Inkluzitron.Models.Settings
+{
+    public class BdsmTestOrgSettings
+    {
+        public string NoResultsOnRecordMessage { get; }
+        public string InvalidFormatMessage { get; }
+        public string LinkAlreadyPresentMessage { get; }
+        public string InvalidTraitMessage { get; }
+        public string InvalidPercentageMessage { get; }
+        public string InvalidTraitCountMessage { get; }
+        public string NoMatchesMessage { get; }
+        public int MaximumMatchCount { get; }
+        public double TraitReportingThreshold { get; }
+        public string NoTraitsToReportMessage { get; }
+        public string InvalidVerboseModeUsage { get; }
+        public string BadFilterQueryMessage { get; }
+        public IReadOnlySet<string> TraitList { get; }
+        public string TestLinkUrl { get; }
+
+        public BdsmTestOrgSettings(IConfiguration config)
+        {
+            const string SectionName = "BdsmTestOrgQuizModule";
+            string GetRequiredConfig(string key)
```

Comment:
Tahle metoda by se krásně dala napsat jako extension metoda nad typem IConfiguration `string GetRequiredValue(this IConfiguration configuration, string key);`